plugins/modules/nmcli.py

```diff
@@ -1738,7 +1803,7 @@ def connection_options(self, detect_change=False):
         return options
 
     @property
-    def ip_conn_type(self):
+    def is_ip_conn_type(self):
```

Comment:
Please note that changing code style for code you are not maintaining is never a good idea without asking first.

Reply:
Agrees this. I should rollback it and do it correct way. Can you describe please, how to determine who maintaining code of one or another module?

Reply:
Usually you can see whom the bot automatically pings when creating a new issue or PR. In the case of nmcli, there is no active maintainer, but there are a lot of individual contributions from several persons.

In general the best approach is to first create an issue explaining what you think would be better, see whether the bot tags someone, then wait to see whether these persons are still active / interested, and if there is no reaction try tagging persons who made changes to that piece of code in the past (and thus seem to be interested in it as well). This is a lengthy process, but it will avoid unnecessary work.

Reply:
Good. Thanks you for explanation!